### ISSUE TITLE
Remove Sections

### ISSUE THREAD

**Issue by zachdaniel** [ash-project/spark#52]
Currently, sections don't really give us much, and ought to be replaceable with Entities.

We can shift our DSL builder to just use entities, and then we just import the top level entity. The only thing we need to support is a config on entities that repeated calls to it merge configs (like sections to).

**Comment by joshprice** [comment]
This would alleviate some confusion, it's hard to explain what the difference really is.

**Comment by zachdaniel** [comment]
Agreed. Unfortunately this would be a massive change that may never make the cut in terms of wether or not its worth it. But yeah it should have just been entities (i.e struct constructors) all the way down.
